### PR TITLE
Pods 8597 member summary fix

### PR DESCRIPTION
**Pull request by PavelVjalicin**
Stubs - https://github.com/hmrc/pension-scheme-event-reporting-stubs/pull/51
FE - https://github.com/hmrc/pension-scheme-event-reporting-frontend/pull/221

### CODE REVIEW

File: app/transformations/ETMPToFrontEnd/API1834Summary.scala [hunk 1]
```diff
@@ -29,41 +29,50 @@ object API1834Summary {
...
-      case _ =>
-        JsSuccess(false)
+  private def memberReads(list:List[EventType]) = {
+    def combineReads(list: List[Reads[Option[Int]]]) = list
+      .foldLeft(Reads.pure(Seq.empty[Option[Int]])) { (acc, reads) =>
+        for {
+          acc <- acc
+          reads <- reads
+        } yield {
+          acc :+ reads
```
[the remark this answers is not in the document]
Comment (PavelVjalicin) [reply app/transformations/ETMPToFrontEnd/API1834Summary.scala:52]: We are accumulating an empty reads and adding one of reads of Option[Int] to each other.
Essentially, transforming Seq[Reads[Option[Int]]] into Reads[Seq[Option[Int]]]

[the remark this answers is not in the document]
Comment (PavelVjalicin) [reply app/transformations/ETMPToFrontEnd/API1834Summary.scala:52]: This adds one of reads of Option[Int] to Seq[Option[Int]] reads.

File: app/transformations/ETMPToFrontEnd/API1834Summary.scala [hunk 2]
```diff
@@ -29,41 +29,50 @@ object API1834Summary {
...
     }
   }
-  private def createRow(event: Option[Int], eventType: String): JsObject = {
+  private def createRow(event: Option[Int], eventType: String, numberOfMembers: Option[Int] = None): JsObject = {
     event.fold(Json.obj())(version =>
       Json.obj(
         "eventType" -> eventType,
         "recordVersion" -> version
-      )
+      ) ++ numberOfMembers.map { x => Json.obj("numberOfMembers" -> JsNumber(x))}.getOrElse(Json.obj())
```
Comment (jamdade-sharad) [inline app/transformations/ETMPToFrontEnd/API1834Summary.scala:41]: please provide meaningful variable name for x

**Review by jamdade-sharad — CHANGES_REQUESTED** [review]
please see comments

File: app/transformations/ETMPToFrontEnd/API1834Summary.scala (hunk 2 above)
Comment (PavelVjalicin) [reply app/transformations/ETMPToFrontEnd/API1834Summary.scala]: Changed.

**Review by jamdade-sharad — APPROVED** [review]

**Pull request merged**